### ISSUE TITLE
Test - Adresses

### ISSUE THREAD

**Issue by georgmaisser** [Wunderbyte-GmbH/moodle-local_shopping_cart#72]
Address is important for taxes.  We have billing & shipping Address. Only billing Address should have effect on taxes etc.

- [x] - Test if different countries get the right VAT according to their respective tax categories.

- [x] - Test if different mode of net prices (added or included, depending on shopping cart settings) work as expected

- [x] - don't test the address form for the moment, just the result. We might remodel the form to a dynamic form. Implementation at the moment is old and not according to Moodle APIs

**Comment by eynimeni** [comment]
see link to screen recordings with details in #71

**Comment by semteacher** [comment]
At the moment the following JS error prevent me from editing addresses
![image](https://github.com/Wunderbyte-GmbH/moodle-local_shopping_cart/assets/1571998/5208e51e-7883-4cd7-840c-22fb626a77c1)

**Comment by georgmaisser** [comment]
Hi Andrii, we didn't refactor the address yet, so no need for further work at the moment. the js issue is not there anymore, I presume?

**Comment by semteacher** [comment]
@georgmaisser  yes, JS error was solved, tests for 3 of 3 cases also alrady existed
